src/components/Create/components/pages/ProjectToken/hooks/ProjectTokenForm.ts

```diff
@@ -45,16 +46,16 @@ export const useProjectTokensForm = () => {
       ? formatIssuanceRate(fundingCycleData.weight)
       : DefaultSettings.initialMintRate
     const reservedTokensPercentage = fundingCycleMetadata.reservedRate
-      ? parseFloat(fundingCycleMetadata.reservedRate)
+      ? fromReduxPercent(fundingCycleMetadata.reservedRate)
```

Comment:
Wanna make mention of this just in case: if we're gonna store percentages in redux, gotta make sure that the data is transposed to the correct units before passing to the `launch` transaction. Using functions like `parseReservedRate` etc.